WcaOnRails/app/views/competitions_mailer/submit_report_nag.html.erb

```diff
@@ -17,5 +17,15 @@
 </p>
 
 <p>
-  Regards, The WCA Board.
+  History:
+  <ul>
+    <% @competition.delegates.each do |d| %>
```

Comment:
We're pretty consistent throughout the website about calling a variable like this `delegate` instead of just `d`.

Reply:
Ups, I'll change this!